apps/www/content/docs/installation/gatsby.mdx

```diff
@@ -52,13 +52,14 @@ Add the code below to the tsconfig.json file to resolve paths:
 }
 ```
 
-### Create gatsby-node.js file
+### Create gatsby-node.ts file
```

Comment:
You can use TS for `gatsby-node`, see https://www.gatsbyjs.com/docs/how-to/custom-configuration/typescript/#gatsby-nodets